annotators/emotion_detection/service_configs/emotion_detection/service.yml

```diff
@@ -0,0 +1,24 @@
+name: emotion-detection
+endpoints:
+- model
+compose:
+  env_file:
+  - .env
+  build:
+    args:
+      SERVICE_PORT: 8025
+      SERVICE_NAME: emotion_detection
+      CUDA_VISIBLE_DEVICES: 0
```

Comment:
if the model needs GPU, then CUDA_VISIBLE_DEVICES: 0 should be given to the environment not to build args

Reply:
Removed from args